### DISCUSSION TITLE
可以登录web端，但是ssh无法登录

### DISCUSSION THREAD

**Discussion by supsupsuperstar** [Boos4721/openwrt#166] [Q&A/答疑解惑]
RT。旁路由模式，刷好系统后web后台可以正常登陆，但是ssh无法登陆，用ttyd登陆提示：
login : can't execute '/usr/bin/zsh': No such file or directory
如果用之前的backup恢复配置又能正常ssh
重新刷回这个backup对应版本的系统也是这样。。。。

额，解决了，没装zsh，装上就好了

**Comment by Boos4721** [comment]
compile zsh by default
